### ISSUE TITLE
uptime doesn't take TZ into account

### ISSUE THREAD

**Issue by dikt1338** [hjelev/rpi-mqtt-monitor#183]
I figured, that the "uptime sensor" uses uptime -s but later adds "Z" (=Zulu =CET) to the timestamp value before sending via mqtt.
According to my testing this leads to wrong values as uptime -s is always local (sorry can't find a man entry to prove that. This is according to some blog posts I found and is confirmed by my own tests)

I changed the script like that - maybe this could be considered for further releases?
```
def check_uptime(format):
    if format == 'timestamp':
        full_cmd = "uptime -s"
        tz_cmd = "date +%z"
        tz_str = subprocess.Popen(tz_cmd, shell=True, stdout=subprocess.PIPE).communicate()[0].decode('utf-8').strip()**
        timestamp_str = subprocess.Popen(full_cmd, shell=True, stdout=subprocess.PIPE).communicate()[0].decode('utf-8').strip()
        timestamp = datetime.strptime(timestamp_str, '%Y-%m-%d %H:%M:%S')
        iso_timestamp = timestamp.isoformat() + tz_str  # Append correct offset to indicate `local` time
```
Basically, I use
` data +%z `
to get the time zone offset of the current installation to send the correct timestamp

**Comment by hjelev** [comment]
Thanks I'll look into this, I have another user reporting simillar issues.

**Comment by JaccovL** [comment]
Count me in on this issue. I have the same ;-)
Thanks for the effort on this project! For me this is a great tool

**Comment by seeloesix** [comment]
I'm having same issue. Thank you for your efforts! Let me know if I can provide any testing, etc.

**Comment by seeloesix** [comment]
@dikt1338 Making these edits corrected it for me, thank you.

**Comment by hjelev** [comment]
added to version 1.1.8

**Comment by dikt1338** [comment]
Works like a charm. Thank you so much for this project!
